backend/entity_routes.py

```
@@ -38,60 +116,61 @@ def _name_similarity(query: str, candidate: str) -> float:
-    confidence: float
 
-@router.post("/search", response_model=EntitySearchResponse)
+# ---------------------------------------------------------------------------
+# Routes
+# ---------------------------------------------------------------------------
+
+@router.post("/search")
 async def search_entity(
     request: EntitySearchRequest,
-    current_user: AuthenticatedUser = Depends(get_current_user)
+    current_user: AuthenticatedUser = Depends(get_current_user),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)